src/tigerbeetle.zig

```diff
@@ -248,6 +248,26 @@ pub const CreateTransfersResult = extern struct {
     }
 };
 
+pub const LookupAccountResult = enum(u32) {
+    ok = 0,
+    no_such_account = 1,
```

Comment:
Perhaps `account_not_found` to match how we have eg `debit_account_not_found`? Ditto for `no_such_transfer`.